src/services/request.service.js

```diff
@@ -135,6 +135,42 @@ const updateRequestStatus = async (requestId, status) => {
   throw new ErrorHandler('Please set status to "approved" or "declined"', 400);
 };
 
+const getAllRequestsHotels = async () => {
+  const {
+    Sequelize, trip, hotel, request,
+  } = db;
+
+  const { Op } = Sequelize;
+
+  const requests = await request.findAll({
+    where: {
+      status: 'approved',
+    },
+    include: [
+      {
+        model: trip,
+        as: 'trips',
+        where: {
+          type: 'return',
```

Comment:
I think you should include both 'one way' and 'return' trips here.

Reply:
Yoiu're right let me implement this.